src/app/app.menus.ts

```diff
@@ -70,4 +70,4 @@ export const isAdminPredicate = (user: User): boolean =>
  * @param user User session data. This will be used to check if the user is an admin
  */
 export const isWorkInProgressPredicate = (user: User): boolean =>
-  environment.production && isAdminPredicate(user);
+  !environment.production && isAdminPredicate(user);
```

Comment:
```suggestion
  !environment.production || isAdminPredicate(user);
```

The predicate is meant to show WIP items if we're in not in production, or if we are, then only admin can see them (so we can test them out on prod but not release to the general public)